fbw-a380x/src/systems/instruments/src/FCU/Components/Heading.tsx

```diff
@@ -1,16 +1,58 @@
 //  Copyright (c) 2023-2024 FlyByWire Simulations
 //  SPDX-License-Identifier: GPL-3.0
 
-import { DisplayComponent, FSComponent, VNode } from '@microsoft/msfs-sdk';
+import {
+  ConsumerSubject,
+  DisplayComponent,
+  EventBus,
+  FSComponent,
+  MappedSubject,
+  SubscribableMapFunctions,
+  VNode,
+} from '@microsoft/msfs-sdk';
+import { FcuEvents } from 'instruments/src/FCU/Publishers/FcuPublisher';
+import { OverheadEvents } from 'instruments/src/MsfsAvionicsCommon/providers/OverheadPublisher';
 
-export interface HeadingProps {}
+export interface HeadingProps {
+  readonly bus: EventBus;
+}
 
 export class Heading extends DisplayComponent<HeadingProps> {
+  private readonly sub = this.props.bus.getSubscriber<FcuEvents & OverheadEvents>();
+
+  private readonly isLightTestActive = ConsumerSubject.create(this.sub.on('ovhd_ann_lt_test_active'), false);
+  private readonly trueRefActive = ConsumerSubject.create(this.sub.on('fcu_push_true_ref'), false);
+
+  private readonly trueIndicatorActive = MappedSubject.create(
+    ([isLightTest, trueRef]) => {
+      if (isLightTest) {
```

Comment:
You could just use an OR here

Reply:
SubscribableMapFunctions.or() would do it.

Reply:
```js
  private readonly trueIndicatorActive = MappedSubject.create(
    SubscribableMapFunctions.or(),
    this.isLightTestActive,
    this.trueRefActive,
  );
```

Like this? @BravoMike99 @tracernz

Reply:
Exactly that.